### PR TITLE
feat: [#37] V2 Initialisation

### PR DESCRIPTION
**Pull request by davdarras**
- new project architecture
  - layout
  - generic api calls
  - clean architecture
- questionnaire CRUD
  - display all questionnaires
  - check pogues identifier and retrieve questionnaire details
  - questionnaire creation
  - questionnaire update (context + survey unit data file input)
  - questionnaire deletion

### CODE REVIEW

File: src/core/factory/NotifierFactory.ts [hunk 1]
```diff
@@ -0,0 +1,4 @@
+import { NotifierPort } from "core/application/port";
+import { useNotifier } from "core/infrastructure";
+
+export const makeNotifier = (): NotifierPort => useNotifier();
```
Comment (laurentC35) [inline src/core/factory/NotifierFactory.ts:4]: You do not need to export this function, in your code, you use directly `useNotifier`, it's good.

**Review by laurentC35 — COMMENTED** [review]
Hello @davdarras , first : well done, the application code is well structured.
I've done some remarks is the code but i've some global remarks !

- Where is the Dockerfile ?
- Can you change logo of app and update manifest.json in public folder ?
- You can replace `<React.Fragment>`by `<>` 😄

**Comment by davdarras** [comment]
>  Where is the Dockerfile ?
    Can you change logo of app and update manifest.json in public folder ?
    You can replace <React.Fragment>by <> 😄

Dockerfile will come afterwards (there's a feature branch for ci integration => feature/ci)

ok for the other remarks

File: README.md [hunk 2]
```diff
@@ -1,3 +1,46 @@
-# Public-Enemy
+# Getting Started with Create React App
```
[the remark this answers is not in the document]
Comment (davdarras) [reply README.md:1]: I'm adding a little summary right now. Pogues, Stromae and queen are not directly dependent for this project, i'll add them on the public enemy backoffice project readme.

File: src/core/factory/QuestionnaireFactory.ts [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+  createQuestionnaireUseCase,
+  QuestionnaireUseCaseType,
+} from "core/application/usecase";
+import { createQuestionnaireRepository } from "core/infrastructure";
+
+/**
+ * Repository factory
+ * @returns questionnaire repository factory
+ */
+export const makeQuestionnaireRepository = (): QuestionnaireRepositoryPort =>
```
[the remark this answers is not in the document]
Comment (davdarras) [reply src/core/factory/QuestionnaireFactory.ts:12]: I'm a little confused about this one. Could you explain me why ?

File: src/core/utils/http/fetcher.ts [hunk 4]
```diff
@@ -0,0 +1,131 @@
...
+) => multipartFetch<ResponseType>(url, "PUT", payload);
+
+/**
+ * Generic simple fetch request
+ * @param url endpoint for request
+ * @param method HTTP method
+ * @param payload
+ * @returns promise of ResponseType
+ */
+const simpleFetch = <ResponseType>(
```
[the remark this answers is not in the document]
Comment (davdarras) [reply src/core/utils/http/fetcher.ts:73]: i don't need this right now, i'll add it if needed :)

**Review by laurentC35 — APPROVED** [review]

**Pull request merged**